### ISSUE TITLE
Review Ticket Items with UOM = EACH or CUYD

### ISSUE THREAD

**Issue by kgilliam3662** [ConceptualSystems/sms-mobile-portable#11]
SMSMobile Review Ticket.  The current layout is the following:

```
BATTERY                                  Net: 0.0
Price: $5.00                           Gross: 0.0
Tier: T1                                Tare: 0.0
Total: $$750.00                       Deduct: 0.0
```
replace the Tier: T1 with    Units: x.x to look like the following:
```
BATTERY                                 Net: 0.0
Price: $5.00                          Gross: 0.0
Units: 150.0                           Tare: 0.0
Total: $$750.00                      Deduct: 0.0
```
I don't believe you allow the Tier to be changes so there is no reason I can think of
to show it.  This way you don't have to worry about the UOM.
But when you click the item to Edit it, the Units field need to be populated with the
units value I sent you via  JSON.  The value should be 150.0 buts its empty.  see pictures below.

![Snap108](https://user-images.githubusercontent.com/34950529/56763036-ee121b80-6766-11e9-90a5-cceb55865a27.jpg)

![Snap109](https://user-images.githubusercontent.com/34950529/56763122-2dd90300-6767-11e9-906a-33b590b0d6c4.jpg)

**Comment by kgilliam3662** [comment]
![Snap109](https://user-images.githubusercontent.com/34950529/56763089-169a1580-6767-11e9-9a24-c0423e7901c7.jpg)

**Comment by moonlightcheese** [comment]
CUYD or EACH

**Comment by moonlightcheese** [comment]
Tier has been replaced with Units.  Displays CUBICYARDS if the UOM is CUYD, otherwise it shows UNITS.
Dialog now populates values for CUYD and EACH types of UOM.

Fixed in SMS Mobile 1.53
